src/drive/actions/index.js

```diff
@@ -93,6 +100,34 @@ export const openFolder = (folderId) => {
   }
 }
 
+export const fetchRecentFiles = () => {
```

Comment:
Yes, it will go to `redux-cozy-client`, but not today ;)

Reply:
Maybe, but it would require switching at least parts of drive to redux-cozy-client, which is way more work than what we planned to do as part of this story. I would also need to dive deeper onto redux-cozy-client internals — not a bad thing, but also not now.

Reply:
C U in a while, crocodile